lib/yatapp/translations_downloader.ex

```diff
@@ -6,7 +6,18 @@ defmodule Yatapp.TranslationsDownloader do
   alias Yatapp.Config
 
   @api_end_point_url "https://api.yatapp.net/api/v1/project/:project_id/:lang/:format"
-
+  @translation_formats %{
```

Comment:
I wonder @JeneaVranceanu if this should not be implicitly send by the server (so we don't have to maintain this in two places) what do you think?

Reply:
I'm confused a bit. You suggest GETting this dictionary from the server instead of having it statically here? If so - that would be definitelly better! New formats can be added dynamically.